tools/policy-gen/generate-policy-helm.sh

```diff
@@ -31,16 +32,22 @@ for policy in "$@"; do
 
   plural=$(yq e '.spec.names.plural' "${policy_crd_file}")
 
-  policies="${policies}${policies:+, }\"${plural}\": {}"
+  policies=${policies}$plural" "
 
 done
 
 # yq_patch preserves indentation and blank lines of the original file
 function yq_patch() {
-  yq '.' "$2" > "$2.noblank"
-  yq eval "$1" "$2" | diff -B "$2.noblank" - | patch -f --no-backup-if-mismatch "$2" -
+  cat '.' "$2" > "$2.noblank"
+  yq eval "$1" "$2" | diff -w -B "$2.noblank" - | patch -f --no-backup-if-mismatch "$2" -
   rm "$2.noblank"
 }
 
-yq_patch '.plugins.policies = {}' "${HELM_VALUES_FILE}"
-yq_patch '.plugins.policies = { '"${policies}"' }' "${HELM_VALUES_FILE}"
+yq_patch "${VALUES_FILE_POLICY_PATH}"' = {}' "${HELM_VALUES_FILE}"
+
+for policy in $policies; do
+  yq_patch "${VALUES_FILE_POLICY_PATH}.${policy}"' = {}' "${HELM_VALUES_FILE}"
+done
+
+#yq_patch "${VALUES_FILE_POLICY_PATH}"' = {}' "${HELM_VALUES_FILE}"
+#yq_patch "${VALUES_FILE_POLICY_PATH}"' = { '"${policies}"' }' "${HELM_VALUES_FILE}"
```

Comment:
this should be gone?

Reply:
this was replaced with for loop, and applying policy one by one. This is because `patch` was applying changes to some random places in Kong Mesh yamls